scripts/sdk/swagger.json

```diff
@@ -5337,6 +5337,165 @@
+                                  "type": "string"
+                                }
+                              }
+                            }
+                          },
+                          "userCount": {
+                            "type": "integer"
+                          }
+                        }
+                      }
+                    },
+                    "userCount": {
+                      "type": "integer"
+                    },
+                    "sharedBaseCount": {
+                      "type": "integer"
+                    }
+                  },
+                  "x-examples": {
+                    "Example 1": {
+                      "projectCount": 1,
+                      "projects": [
+                        {
+                          "tableCount": {
+                            "table": 3,
+                            "view": 0
+                          },
+                          "viewCount": {
+                            "formCount": 0,
+                            "gridCount": 3,
+                            "galleryCount": 0,
+                            "kanbanCount": 0,
+                            "total": 3,
+                            "sharedFormCount": 0,
+                            "sharedGridCount": 0,
+                            "sharedGalleryCount": 0,
+                            "sharedKanbanCount": 0,
+                            "sharedTotal": 0,
+                            "sharedLockedCount": 0
+                          },
+                          "webhookCount": 0,
+                          "filterCount": 0,
+                          "sortCount": 0,
+                          "rowCount": [
+                            {
+                              "TotalRecords": "76"
+                            }
+                          ],
+                          "userCount": 1
+                        }
+                      ],
+                      "userCount": 1,
+                      "sharedBaseCount": 0
+                    }
+                  }
+                },
+                "examples": {
+                  "example-1": {
+                    "value": "{\n  \"projectCount\": 1,\n  \"projects\": [\n    {\n      \"tableCount\": {\n        \"table\": 3,\n        \"view\": 0\n      },\n      \"viewCount\": {\n        \"formCount\": 0,\n        \"gridCount\": 3,\n        \"galleryCount\": 0,\n        \"kanbanCount\": 0,\n        \"total\": 3,\n        \"sharedFormCount\": 0,\n        \"sharedGridCount\": 0,\n        \"sharedGalleryCount\": 0,\n        \"sharedKanbanCount\": 0,\n        \"sharedTotal\": 0,\n        \"sharedLockedCount\": 0\n      },\n      \"webhookCount\": 0,\n      \"filterCount\": 0,\n      \"sortCount\": 0,\n      \"rowCount\": [\n        {\n          \"TotalRecords\": \"76\"\n        }\n      ],\n      \"userCount\": 1\n    }\n  ],\n  \"userCount\": 1,\n  \"sharedBaseCount\": 0\n}"
```

Comment:
may double check to see if it's outdated.

Reply:
Updated now :+1: